### PR TITLE
Add Multiscale Topographic Position dataset to External data

### PR DESCRIPTION
**Pull request by robbibt**
* [x] I have spellchecked my content using Microsoft Word, Grammarly, or otherwise.
* [ ] If required, update the [DEA Tech Alerts and Changelog][TechAlertsChangelog] and the [DEA Sandbox Updates banner][SandboxUpdatesBanner].

[TechAlertsChangelog]: https://github.com/GeoscienceAustralia/dea-knowledge-hub/blob/main/docs/tech-alerts-changelog/_tech_alerts_changelog.md
[SandboxUpdatesBanner]: https://bitbucket.org/geoscienceaustralia/datakube-apps/src/64c28bbf3d0e019d8940547a22f78b9bfd58d739/clusters/dea-sandbox/sandbox.yaml

### CODE REVIEW

File: docs/data/external-data/ga-mtpi/_overview.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## About
+
+A Multi-scale topographic position image of Australia has been generated by combining a topographic position index and topographic ruggedness. Topographic Position Index (TPI) measures the topographic slope position of landforms. Ruggedness informs on the roughness of the surface and is calculated as the standard deviation of elevations. The derived ternary image captures variations in topographic position across these spatial scales (blue local, green intermediate and red regional) and gives a rich representation of nested landform features that have broad application in understanding geomorphological and hydrological processes and in mapping regolith and soils over the Australian continent.
```
Comment (benji-glitsos-ga) [inline docs/data/external-data/ga-mtpi/_overview.md:3]:
```suggestion
A multi-scale topographic position image of Australia has been generated by combining a topographic position index with topographic ruggedness. The Topographic Position Index (TPI) measures the topographic slope position of landforms. Togographic ruggedness informs on the roughness of the surface and is calculated as the standard deviation of elevations. The derived ternary image captures variations in topographic position across these spatial scales (blue local, green intermediate, and red regional) and gives a rich representation of nested landform features that have broad application in understanding geomorphological and hydrological processes and in mapping regolith and soils over the Australian continent.
```

Comment (robbibt) [reply docs/data/external-data/ga-mtpi/_overview.md:3]: I'm fine with this, but I think it comes down to whether we want to adopt the external provider's wording/content exactly as they have written it with no further modifications (e.g. copy-paste content from their portal), or whether we want to customise it on our end.

Comment (benji-glitsos-ga) [reply docs/data/external-data/ga-mtpi/_overview.md:3]: Oh.. I didnt realise it was a copy-paste. In that case, I'll leave it as-is. I think it's ok to: A) substantially rewrite it, B) keep it exactly as-is.

**Review by benji-glitsos-ga — APPROVED** [review]

**Comment by benji-glitsos-ga** [comment]
Converted to draft because I'll need to get approval for this before pushing it live.

**Comment by robbibt** [comment]
Sounds good, would it be better for us to batch a few of these rather than doing them in individual PRs? (e.g. I could add additional datasets here)

**Comment by benji-glitsos-ga** [comment]
@robbibt I think individual PRs works well. Whatever works for you. I'll probably send them to directorate for approval as a batch

**Comment by benji-glitsos-ga** [comment]
Preview - https://pr-259-preview.khpreview.dea.ga.gov.au/data/external-data/ga-mtpi/

**Pull request merged**